examples/reactive-blinky.rs.outdated

```diff
@@ -1,61 +1,47 @@
 //! Blinks an LED
 
-#![feature(proc_macro)]
 #![deny(unsafe_code)]
 #![deny(warnings)]
+#![no_main]
```

Comment:
same here

Reply:
This file can't be compiled. SysTick exception is missing. I don't know why.
https://github.com/japaric/cortex-m-rtfm/issues/101

Reply:
Maybe it can be updated to use a normal timer?

Reply:
Of course, we can. But I think this is not the goal of this PR.